internal/impl/packages.go

```diff
@@ -86,6 +87,19 @@ func (d *Devbox) Add(ctx context.Context, pkgsNames ...string) error {
 		addedPackageNames = append(addedPackageNames, packageNameForConfig)
 	}
 
+	for _, pkg := range addedPackageNames {
+		if platform != "" {
+			if err := d.cfg.Packages.AddPlatform(pkg, platform); err != nil {
+				return err
+			}
+		}
+		if excludePlatform != "" {
+			if err := d.cfg.Packages.ExcludePlatform(pkg, excludePlatform); err != nil {
+				return err
+			}
+		}
+	}
+
```

Comment:
Can this be part of `d.cfg.Packages.Add(packageNameForConfig)` ?

Reply:
This is a good idea. I'll do it in a stack PR (#1363) to minimize conflicts.

Reply:
actually, moving this won't work.

We need this here to enable the following scenario:
```
devbox add hello --platform x86_64-linux
devbox add hello --platform x86-64-darwin
```

The second time it runs, it willl not be within the loop above where .Add() is called.